streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala

```diff
@@ -176,7 +176,7 @@ class ReceiverSuite extends TestSuiteBase with Timeouts with Serializable {
       blockGenerator.addData(count)
       generatedData += count
       count += 1
-      Thread.sleep(1)
+      Thread.sleep(0)
```

Comment:
Yeah at best it is equivalent to `Thread.yield()` but that wouldn't guarantee any particular behavior in this case. The thread may or may not yield, and other threads may or may not progress at all.